### PR TITLE
Restore error messages from the worker threads

### PR DESCRIPTION
**Pull request by iwillig**

### CODE REVIEW

File: src/worker_broker.js [hunk 1]
```diff
@@ -275,6 +275,7 @@ function setupWorkerThread () {
         }
         catch(e) {
             // Thrown errors will be passed back (in string form) to main thread
+            console.error(e);
```
Comment (bcamper) [inline src/worker_broker.js]: Did you mean to keep this here?

File: src/gl/shader_sources.js [hunk 2]
```diff
@@ -1,22 +1,6 @@
 // Generated from GLSL files, don't edit!
```
Comment (bcamper) [inline src/gl/shader_sources.js]: Weird this file changed, I think it just re-compiled in a different order this time? There aren't/shouldn't be any changes as far as I know.

**Comment by bcamper** [comment]
Sweet!

File: src/worker_broker.js (hunk 1 above)
Comment (iwillig) [inline src/worker_broker.js]: No, good catch.

File: src/gl/shader_sources.js (hunk 2 above)
Comment (iwillig) [inline src/gl/shader_sources.js]: Yeah that is strange... let me try to fix it.

**Comment by iwillig** [comment]
Okay fixed the sharder source file

**Pull request merged**